core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -1181,6 +1421,30 @@ short CmpSeabaseDDL::createSeabaseTable2(
       return -1;
     }
 
+  // If creating an external table, go perform operation
+  if (createTableNode->isExternal())
+    {
+      retcode = createSeabaseTableExternalHive(cliInterface, createTableNode, currCatName, currSchName);
```

Comment:
if this function returns -2, should we perform a cleanup operation of the meta-data tables here?